### PR TITLE
[ReadStat] Version bump and apple silicon

### PR DESCRIPTION
**Pull request by jeremiahpslewis**

### CODE REVIEW

**Comment by junyuan-chen** [comment]
@vchuravy @davidanthoff

Could someone take a look at this and merge the PR?

File: R/ReadStat/build_tarballs.jl [hunk 1]
```diff
@@ -22,9 +23,12 @@ fi
 # Windows doesn't search ${prefix}/include?
 export CPPFLAGS="${CPPFLAGS} -I${prefix}/include"
 
+# Fix "Undefined symbols for architecture arm64: "_libiconv","
+export LDFLAGS="-L${libdir}"
+
 cd $WORKSPACE/srcdir/ReadStat/
 ./autogen.sh
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} "${FLAGS[@]}"
```
Comment (giordano) [inline R/ReadStat/build_tarballs.jl:31]: ?

Comment (junyuan-chen) [reply R/ReadStat/build_tarballs.jl:31]: Does it have something to do with libiconv on Mac Apple silicon that used to give an error?

Comment (jeremiahpslewis) [reply R/ReadStat/build_tarballs.jl]: Snippet was unnecessary. Good to go now.

**Pull request merged**